view/frontend/web/insights/add-to-cart-mixin.js

```diff
@@ -0,0 +1,51 @@
+define(['jquery'], function ($) {
+    'use strict';
+
+    var algoliaAddToCartMixin = {
+
+        submitForm: function (form) {
+            if (window.algoliaConfig
+                && algoliaConfig.ccAnalytics.enabled
+                && algoliaConfig.ccAnalytics.conversionAnalyticsMode == 'place_order'
+            ) {
+                this._setQueryIdToForm(form)
+            }
+
+            this._super(form);
+        },
+
+        _setQueryIdToForm: function(form) {
+            if (form.find('button[type="submit"]').length
```

Comment:
is it really `find` ?
I would have expected to have hear `form.querySelector()` instead. Although I can't really tell if form is a DOM element. Is it?

Reply:
It's a jQuery element at this point of the mixin for `mage.catalogAddToCart`.